test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -520,15 +547,42 @@ Address RandomCodeOptions::getRandomAddress(AddressType _type) const
 	{
 		case AddressType::Precompiled:
 			return precompiledAddressList[(int)RandomCode::randomUniInt(0, precompiledAddressList.size())];
+		case AddressType::DestinationAccount:
+			if (RandomCode::randomPercent() < emptyAddressProbability)
+				return ZeroAddress;
```

Comment:
I think we should add a comment that zeroAddress means contract creation if it is a transaction